source/ColorWindow.cpp

```diff
@@ -97,16 +105,18 @@ void ColorWindow :: MessageReceived (BMessage* message) {
 			}
 			break;
 
-		// It answer to an UNDO request
-		case BUTTON_DEFAULT: {
-
-			alert = new BAlert("", B_TRANSLATE("Set default values ?"), B_TRANSLATE("Yes"), B_TRANSLATE("No"), NULL, B_WIDTH_AS_USUAL, B_WARNING_ALERT);
-			alert->SetShortcut(0, B_ESCAPE);
+		// It answer to an REVERT request
+		case BUTTON_REVERT: {
+			fColorControl->SetValue(gBgColor);
+			this->PostMessage(COLOR_CHANGED);
+		}
+		break;
 
-			if (alert->Go() == 0) {
-				fColorControl->SetValue(gBgColor);
-				this->PostMessage(COLOR_CHANGED);
-			}
+		// It answer to SET AS DEFAULT request
+		case SET_DEFAULT_COLOR: {
+			msg = new BMessage(SET_DEFAULT_COLOR);
+			msg -> AddColor("new_def_color", fColorControl -> ValueAsColor());
```

Comment:
No spaces before and after "->"
There are some more places that need this style change.

Reply:
Oh, I find it better with spaces. Nevertheless, changed about 200 " -> " to "->"